src/services/codefixes/fixMissingTypeAnnotationOnExports.ts

```diff
@@ -504,6 +505,12 @@ function createChainedExpression(expression: ExpressionReverseChain, expressionT
 }
 
 function typeToTypeNode(type: Type, enclosingDeclaration: Node, typeChecker: TypeChecker) {
+    if (type.flags & TypeFlags.UniqueESSymbol) {
+        return factory.createTypeOperatorNode(SyntaxKind.UniqueKeyword, factory.createKeywordTypeNode(SyntaxKind.SymbolKeyword));
+    }
+    else if (type.flags & TypeFlags.ESSymbolLike) {
+        return factory.createKeywordTypeNode(SyntaxKind.SymbolKeyword);
+    }
     const typeNode = typeChecker.typeToTypeNode(
         type,
         enclosingDeclaration,

```

Comment:
I don't think us handling symbols here is a good approach. The emitter uses `createTypeOfDeclaration` maybe we can't use that directly, but `typeToTypeNode` should be able to print `unique symbol` if we pass it `NodeBuilderFlags.AllowUniqueESSymbolType` flags:

```ts
function createTypeOfDeclaration(declarationIn: AccessorDeclaration | VariableLikeDeclaration | PropertyAccessExpression, enclosingDeclaration: Node, flags: NodeBuilderFlags, tracker: SymbolTracker, addUndefined?: boolean) {
    const declaration = getParseTreeNode(declarationIn, isVariableLikeOrAccessor);
    if (!declaration) {
        return factory.createToken(SyntaxKind.AnyKeyword) as KeywordTypeNode;
    }
    // Get type of the symbol if this is the valid symbol otherwise get type at location
    const symbol = getSymbolOfDeclaration(declaration);
    let type = symbol && !(symbol.flags & (SymbolFlags.TypeLiteral | SymbolFlags.Signature))
        ? getWidenedLiteralType(getTypeOfSymbol(symbol))
        : errorType;
    if (type.flags & TypeFlags.UniqueESSymbol &&
        type.symbol === symbol) {
        flags |= NodeBuilderFlags.AllowUniqueESSymbolType;
    }
    if (addUndefined) {
        type = getOptionalType(type);
    }
    return nodeBuilder.typeToTypeNode(type, enclosingDeclaration, flags | NodeBuilderFlags.MultilineObjectLiterals, tracker);
}
```

Reply:
I actually didn't know that was the issue. Fixed.